wp-content/themes/bzLLKit/functions.php

```diff
@@ -1053,9 +1053,10 @@ function bz_attendance($atts, $content = null) {
     if($width < 20)
        $width = 100;
 
-    $what = str_replace("&#038;", "&", get_the_title());
```

Comment:
This was an earlier hack for the same issue, but with ampersands. I'm moving it to the attendance code itself and trying to handle it a bit more robustly.